app/actions/views/edit_profile.js

```diff
@@ -25,7 +25,21 @@ export function setProfileImageUri(imageUri = '') {
     };
 }
 
+export function removeProfileImage(user, success, error) {
```

Comment:
Why using callbacks here?

Reply:
I may not fully grasp the underlying redux action, but the 'updateUser' action on edit_profile.js line 8 is using the callbacks as well.